templates/firefox-ios.xliff

```diff
@@ -3387,6 +3441,103 @@ Private Tab</source>
+        <note>The text for the clear cookies and site data alert button inside the enhanced tracking protection screen.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.ClearData.AlertText.v128" xml:space="preserve">
+        <source>Removing cookies and site data for %@ might log you out of websites and clear shopping carts.</source>
+        <note>The text for the clear cookies and site data alert inside the enhanced tracking protection screen.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.ClearData.AlertTitle.v128" xml:space="preserve">
+        <source>Clear cookies and site data</source>
+        <note>The title for the clear cookies and site data alert inside the enhanced tracking protection screen.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.ClearData.ButtonTitle.v128" xml:space="preserve">
+        <source>Clear cookies and site data</source>
+        <note>The title for the clear cookies and site data button inside the enhanced tracking protection screen.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.ClearData.ToastMessage.v128" xml:space="preserve">
+        <source>Cookies and site data removed</source>
+        <note>The text for the clear cookies and site data toast that appears when the user selects to clear the cookies</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.ConnectionSecure.v128" xml:space="preserve">
+        <source>Secure connection</source>
+        <note>Text to let users know that the current website is secure.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.ConnectionUnsecure.v128" xml:space="preserve">
+        <source>Connection not secure</source>
+        <note>Text to let users know that the current website is not secure.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.Trackers.CrossSite.v128" xml:space="preserve">
+        <source>%@ Cross-site tracking cookies</source>
+        <note>Text to let users know how many cross-site tracking cookies were blocked on the current website.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.Trackers.Fingerprinter.v128" xml:space="preserve">
+        <source>%@ Fingerprinter</source>
+        <note>Text to let users know how many fingerprinters were blocked on the current website.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.Trackers.SocialMedia.v128" xml:space="preserve">
+        <source>%@ Social media tracker</source>
+        <note>Text to let users know how many social media trackers were blocked on the current website.</note>
+      </trans-unit>
+      <trans-unit id="Menu.EnhancedTrackingProtection.Details.Trackers.v128" xml:space="preserve">
+        <source>%@ Trackers blocked</source>
```

Comment:
Please explain what the placeholder means in localization comments, so translators know the context and can adapt their translations.
Another thing: unfortunately we don't yet handle plurals properly. A workaround for this string would be "Trackers blocked: %@"

Reply:
See limitations in our system at the bottom called out [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms).

Reply:
I can see the plural string updated for tackers as requested. Do we need to make same changes for social media, fingerprinters etc?

Reply:
@tusharC95 If they are formatted like this, then yes, correct :) thanks for checking in about that